src/dragdrop.js

```diff
@@ -0,0 +1,68 @@
+import localStorageConst from './localstorage';
+
+let dragSrcEl;
+function createNewList() {
+  const newListElements = document.getElementsByTagName('li');
+  const newList = [];
+  for (let i = 0; i < newListElements.length; i += 1) {
+    const newItem = {};
+    const title = newListElements[i].textContent;
+    newItem.description = title;
+    const index = newListElements[i].getAttribute('index');
+    newItem.index = index;
+    const completed = newListElements[i].getAttribute('completed');
+    if (String(completed) === 'true') {
+      newItem.completed = true;
+    } else {
+      newItem.completed = false;
+    }
+    newList.push(newItem);
+  }
```

Comment:
Using `forEach` method instead of using for loops makes code shorter and cleaner.